faculty/clients/job.py

```diff
@@ -57,6 +67,26 @@ class RunState(Enum):
     ["name", "description", "author_id", "created_at", "last_updated_at"],
 )
 JobSummary = namedtuple("JobSummary", ["id", "metadata"])
+InstanceSize = namedtuple("InstanceSize", ["milli_cpus", "memory_mb"])
+JobParameter = namedtuple(
+    "JobParameter", ["name", "type", "default", "required"]
+)
+JobCommand = namedtuple("JobCommand", ["name", "parameters"])
+JobDefinition = namedtuple(
+    "JobDefinition",
+    [
+        "working_dir",
+        "command",
+        "image_type",
+        "conda_environment",
+        "environment_ids",
+        "instance_size_type",
+        "instance_size",
+        "max_runtime_seconds",
+    ],
+)
+Job = namedtuple("Job", ["job_id", "meta", "definition"])
```

Comment:
Our convention (as you can see in `JobSummary` and the run / subrun namedtuples) is to have the primary UUID for an object be just `id` Can you rename the `job_id` attribute here to just `id` (and update the schemas and tests accordingly).